locations/MM.json

```diff
@@ -4,7 +4,8 @@
         "chest_unopened_img": "/images/items/closed_Chest.png",
         "chest_opened_img": "/images/items/open_Chest.png",
         "access_rules":[
-            "@MM/Misery Mire"
+            "@MM/Misery Mire, boots",
+            "@MM/Misery Mire, hookshot"

```

Comment:
I just thought of another way of dchekcing that. 
would  you add an access_rule at the darkworld.json MM/Misery Mire section to check for boots or hookshot there